config/deploy.rb

```diff
@@ -0,0 +1,44 @@
+# config valid for current version and patch releases of Capistrano
+lock "~> 3.10.1"
+
+# set :application, "my_app_name"
+# set :repo_url, "git@example.com:me/my_repo.git"
+
+set :application, "heroes_team_app"
+set :repo_url, "git@github.com:Ruby-IF081/heroes_team_app.git"
+
+# Default branch is :master
+ask :branch, :master # `git rev-parse --abbrev-ref HEAD`.chomp
+
+# Default deploy_to directory is /var/www/my_app_name
+# set :deploy_to, "/var/www/my_app_name"
+set :deploy_to, "/var/www/heroes_team_app/code"
```

Comment:
are you sure, that you want deploy there?

Reply:
sure, already

Reply:
done

Reply:
I believe this now was not about deploy.rb/production.rb but it was about deployment path.
Why did you add ```/code``` directory?

Reply:
just followed the one of tutorial partly, and they used such directory structure

Reply:
@egorodessa please remove that ```/code```

Reply:
done